src/setupTests.ts

```diff
@@ -0,0 +1,6 @@
+import { configure } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+
+configure({ adapter: new Adapter() });
```

Comment:
From where is this setup called? 
Is this automatically called by Jest?

Reply:
yes. Check out the [dokumentation](https://facebook.github.io/create-react-app/docs/running-tests#src-setuptestsjs)